test/CrateTest/PDO/PDOTest.php

```diff
@@ -120,36 +119,26 @@ public function testInstantiationWithHttpAuth() {
      */
     public function testSetAttributeWithHttpBasicAuth()
     {
-        $user = 'crate';
-        $passwd = 'secret';
-        $expectedCredentials = [$user, $passwd];
-
-        $client = $this->getMock(ClientInterface::class);
-        $pdo = new PDO('crate:localhost:44200', null, null, []);
-        $reflection = new ReflectionClass(PDO::class);
-        $property = $reflection->getProperty('client');
-        $property->setAccessible(true);
-        $property->setValue($pdo, $client);
-        $client
-            ->expects($this->once())
-            ->method('setHttpBasicAuth')
-            ->with($user, $passwd);
-
-        $pdo->setAttribute(PDO::CRATE_ATTR_HTTP_BASIC_AUTH, [$user, $passwd]);
-        $this->assertEquals($expectedCredentials, $pdo->getAttribute(PDO::CRATE_ATTR_HTTP_BASIC_AUTH));
+        $this->markTestIncomplete('Missing implementation');
```

Comment:
Was this agreed upon?

Reply:
I forgot to remove it since it's already tested else where.